tools/js-sdk-release-tools/src/mlc/clientGenerator/modularClientPackageGenerator.ts

```diff
@@ -78,6 +79,7 @@ export async function generateAzureSDKPackage(options: ModularClientPackageOptio
         logger.error(`Failed to generate package due to ${(err as Error)?.stack ?? err}`);
         throw err;
     } finally {
+        await codeOwnersAndIgnoreLinkGenerator(options.typeSpecDirectory);
```

Comment:
move to line 30. dont put function that has chance to throw exception in finally block.
you can pass in `packageDirectory` which should be the generated sdk folder path